### PR TITLE
Add support for Askey STI6130

### PR DESCRIPTION
**Pull request by prabhjotsbhatia-ca**
The Askey STI6130 is a widely used box sold under various brands including Bell Streamer in Canada, Google ADT-3 Developer box, Dynalink etc. and is based on the Amlogic S905Y2 board, which is the same board as the widely hyped Walmart Onn android tv box. 

I don't have an Onn box to test, but it is likely this same scheme would work on that as well. 

Tested: 
- Current app detection ✅ 
- Play/Pause/standby/idle states ✅ 
- Tested consistency across various apps. 
All unit tests pass. 

I had to add a new key to detect the product ID.

### CODE REVIEW

File: androidtv/basetv/basetv.py [hunk 1]
```diff
@@ -167,6 +167,14 @@ def _cmd_current_app(self):
         ):
             return constants.CMD_CURRENT_APP_GOOGLE_TV
 
+        # Is this an Askey STI6130 Device?
+        if (
+            self.DEVICE_ENUM == constants.DeviceEnum.ANDROIDTV
+            and "askey" in self.device_properties.get("manufacturer")
+            and "sti6130" in self.device_properties.get("product_id")
```
Comment (JeffLIrion) [inline androidtv/basetv/basetv.py:174]:
```suggestion
            and "askey" in self.device_properties.get("manufacturer", "")
            and "sti6130" in self.device_properties.get("product_id", "")
```

File: androidtv/basetv/basetv.py [hunk 2]
```diff
@@ -422,11 +439,11 @@ def _parse_device_properties(self, properties):
             return
 
         lines = properties.strip().splitlines()
-        if len(lines) != 4:
+        if len(lines) != 5:
             self.device_properties = {}
             return
 
-        manufacturer, model, serialno, version = lines
+        manufacturer, model, serialno, version, product_id = lines
```
Comment (JeffLIrion) [inline androidtv/basetv/basetv.py:446]: Please add `product_id` in the documentation for this function.

File: androidtv/basetv/basetv.py (hunk 1 above)
Comment (prabhjotsbhatia-ca) [reply androidtv/basetv/basetv.py:174]: Thanks for reviewing! Committed your suggestion.

**Comment by prabhjotsbhatia-ca** [comment]
@JeffLIrion thank you for your comments! I was able to fix linting and added the product_id to the  documentation. 

Additionally, I found an existing bug in the documentation for the _parse_device_properties function here: 

https://androidtv.readthedocs.io/en/stable/androidtv.basetv.basetv.html#androidtv.basetv.basetv.BaseTV._parse_device_properties

(The "Returns" section header is missing in the code and consequently all the documentation is showing up incorrectly under Parameters) 

Would you like me to fix that as a part of this PR or would you like me to open a subsequent PR after this current one is merged?

**Pull request merged**